integration/update_field_compat_test.go

```diff
@@ -54,6 +58,86 @@ func TestUpdateFieldCompatInc(t *testing.T) {
 	testUpdateCompat(t, testCases)
 }
 
+func TestUpdateFieldCompatMax(t *testing.T) {
+	t.Parallel()
+	setup.SkipForTigrisWithReason(t, "tigris does not support dynamic types")
```

Comment:
What does it mean and why we skip all tests for Tigris? We should not do that

Reply:
When you specify a max value in a type other than existing in database, it will compare them and overwrite based on type comparison order. In tigris, as far as I know we cannot set the value with different type than existing (i.e. types are not dynamic)

Reply:
So we can't test that with Tigris at all and have to skip all tests?

Reply:
Definitely there should be some basic tigris testing. Although I'm not sure if there's any way to modify compat tests to exclude some collections for specified test

Reply:
https://github.com/FerretDB/FerretDB/blob/be84709aa9d9363bf566485a25ea34006e9aa223/integration/update_compat_test.go#L128-L133

Those tests should already be skipped by that code

Reply:
Thanks, it works!